list_manip.py

```diff
@@ -0,0 +1,81 @@
+def list_manipulation(list, command, location, value=None):
+    """Mutate lst to add/remove from beginning or end.
+
+    - lst: list of values
+    - command: command, either "remove" or "add"
+    - location: location to remove/add, either "beginning" or "end"

```

Comment:
The instructions say to use strings as args, but a more robust/commonly-used way of having string args is with [enumerations](https://docs.python.org/3/library/enum.html#creating-an-enum). Prevents typos and makes it easier for user to know their options.
```python
from enum import Enum, auto

class Command(Enum):
    add = auto()
    remove = auto()

class Location(Enum):
    beginning = auto()
    end = auto()
```
And would be used like
```python
list_manipulation([1,2,3], Command.add, Location.end, 4)
```